### DISCUSSION TITLE
What does -pi do?

### DISCUSSION THREAD

**Discussion by CaledoniaProject** [projectdiscovery/nuclei#2413] [Q&A] [answered]
I'm wondering what is an "internal request"?

**Comment by ehsandeep** [comment, marked as answer]
@CaledoniaProject `-proxy-internal` includes non template HTTP requests as well such as version check api, 3rd party [reporting](https://nuclei.projectdiscovery.io/nuclei/get-started/#nuclei-reporting) apis etc.
